src/components/atom/AppBurgerMenuButton.tsx

```diff
@@ -2,7 +2,7 @@ import BurgerMenuIcon from "../icons/BurgerMenuIcon";
 
 const AppBurgerMenuButton = () => {
     return (
-        <button className="bg-white h-12 w-12 rounded-full flex justify-center items-center app-shadow">
+        <button className="flex h-12 w-12 items-center justify-center rounded-full bg-white app-shadow">
```

Comment:
It's probably work in progress but I'd love to have a global style for app buttons so in code we will have something like "button-background" or "button-background burguer" or whatever.